src/schema/objects/suffixes.yaml

```
@@ -286,16 +292,23 @@ RB1map:
   unit: arbitrary
 S0map:
   value: S0map
-  display_name: Observed signal amplitude (S0) image
+  display_name: Projected baseline signal amplitude (S0) image
   description: |
     In arbitrary units (arbitrary).
+    This represents a model-based estimate of signal intensity
+    in the absence of the contrast under investigation.
     For a multi-echo (typically fMRI) sequence, S0 maps index the baseline signal
     before exponential (T2-star) signal decay.
```

```suggestion
    For a multi-echo sequence
    (eg. multi-echo spin echo T2 relaxometry or multi-echo fMRI),
    S0 maps index the baseline signal at a hypothetical echo time of zero
    (that is, in the absence of any T2 or T2-star decay).
```

Guessing from fact that `S0map` is a valid suffix for anatomical MRI that it's there for the context of T2 relaxometry? In which case that's arguably a *better* exemplar than multi-echo fMRI, where such a parametric map is much more likely to come from a processing pipeline than the scanner.